libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -181,15 +185,21 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_aztec_editor, container, false);
 
-        // request dependency injection
-        if (getActivity() instanceof EditorFragmentActivity) {
-            ((EditorFragmentActivity) getActivity()).initializeEditorFragment();
-        }
-
         title = (EditTextWithKeyBackListener) view.findViewById(R.id.title);
         content = (AztecText) view.findViewById(R.id.aztec);
         source = (SourceViewEditText) view.findViewById(R.id.source);
 
+        // Set the default value for max and min picture sizes.
+        maxImageWidthForVisualEditor = ImageUtils.getMaximumThumbnailWidthForEditor(getActivity());
+        minImageWidthForVisualEditor = DisplayUtils.dpToPx(getActivity(), MIN_BITMAP_DIMENSION_DP);
```

Comment:
This is a nitpick and not direct part of the changes here, but looks relevant: what do you think about modifying `ImageUtils.getMaximumThumbnailWidthForEditor() ` to receive `MIN_BITMAP_DIMENSION_DP` as a parameter along with the `Context`, and use it instead the `48` hardcoded value in there?